content/sites/ignore-assets.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Ignoring Subsets of Static Assets
+alwaysopen: true
+weight: 5
+---
+
+Worker Sites require [Wrangler](https://github.com/cloudflare/wrangler) - make sure to be on the [latest version](/quickstart/#updating-the-cli) - and the Workers [Unlimited plan](https://workers.cloudflare.com/sites#plans).
+
+There are cases where users may not want to upload certain static assets to their Workers Sites.
+In this case, Workers Sites can also be configured to ignore certain files or directories using logic 
+similar to [Cargo's optional include and exclude fields](https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields-optional).
+This means that we use gitignore semantics when declaring which directory entries to include or ignore in uploads.
+
+### Exclusively including files/directories
+
+If you want to include only a certain set of files or directories in your `bucket`, you can add an `include` field to your
+`[site]` section of `wrangler.toml`:
+```toml
+[site]
+bucket = "./public"
+entry-point = "workers-site" 
+include = ["included_dir"] # must be an array.
+```
+Wrangler will only upload files or directories matching the patterns in the `include` array.
+
+### Excluding files/directories
+
+If you want to exclude files or directories in your `bucket`, you can add an `exclude` field to your
+`[site]` section of `wrangler.toml`:
+```toml
+[site]
+bucket = "./public"
+entry-point = "workers-site" 
+exclude = ["excluded_dir"] # must be an array.
+```
+Wrangler will ignore files or directories matching the patterns in the `exclude` array when uploading assets to Wrangler.
+
+### Include > Exclude
+
+If you provide both `include` and `exclude` fields, the `include` field will be used and the `exclude` field will be ignored.
```

Comment:
Is this what we want? I think I missed this part of the implementation and it probably makes sense/is convention but... I can see a world where we have an `include: ["dir/img"]` and an `exclude: ["dir/img/way-too-big.jpg"]` and you'd wanna use that.

Reply:
what you're suggesting is not consistent with cargo `include/exclude` behavior; the behavior above is!

Reply:
cool :)